lib/y2lan_restart_common.pm

```diff
@@ -421,7 +421,7 @@ Close yast2 lan configuration and check that it is closed successfully
 
 =cut
 sub close_yast2_lan {
-    send_key "alt-o";    # OK=>Save&Exit
+    wait_screen_change { send_key "alt-o" };    # OK=>Save&Exit
```

Comment:
in the case of the failure we send `alt-o` but the SUT doesn't processed, otherwise we would see the next screen after the failure here: https://openqa.suse.de/tests/8815455#step/yast2_lan_device_settings/40
Waiting after sending it will not help this sporadic issue, we need to ensure that the OK is pressed.